packages/tree-view/src/__stories__/stories.js

```diff
@@ -21,6 +23,30 @@ import {
 import { TreeItem } from "../index";
 import sampleTreeNodeObject from "../__fixtures__/sampleTreeNodeObject";
 
+// test that the TreeView can render a high-order TreeItem component or a function TreeItem component
+export class HocTreeItem extends React.Component {
+  render() {
+    return <TreeItem {...this.props}>{this.props.children}</TreeItem>;
+  }
+}
+HocTreeItem.propTypes = {
+  /**
+   * Accepts other high order TreeItem components
+   */
+  children: PropTypes.node
+};
+
+export const FuncTreeItem = props => (
```

Comment:
Same thing here, can we find a word or series of full words to describe this item?